driver.go

```diff
@@ -586,7 +587,18 @@ func createConnector(d *Driver, connectorConfig ConnectorConfig) (*connector, er
 		connectorConfig.Configurator(&config, &opts)
 	}
 	if connectorConfig.AutoConfigEmulator {
-		if err := autoConfigEmulator(context.Background(), connectorConfig.Host, connectorConfig.Project, connectorConfig.Instance, connectorConfig.Database); err != nil {
+		if connectorConfig.Host == "" {
+			connectorConfig.Host = "localhost:9010"
+		}
+		schemeRemoved := regexp.MustCompile("^(http://|https://|passthrough:///)").ReplaceAllString(connectorConfig.Host, "")
```

Comment:
I should like to use url.Parse then update the url scheme instead.

Reply:
nvm, replacing using url.Parse is more safe with the URL, also handle the case sensitive. But if other places using the same approach, keep it consistent